reactfrontend/src/components/addAssetForm/addAssetForm.js

```diff
@@ -94,7 +100,12 @@ class AddAssetForm extends Component {
     event.preventDefault()
     const newData = {...this.state.asset}
     let newDataElement = {...newData[identifier]}
-    newDataElement = event.target.value
+    console.log(identifier)
```

Comment:
Is a 'console.log' what was actually intended in this case?

Reply:
it was necessary for developing purposes and nothing else

Reply:
Thank you for clarifying. I hope the comments are useful to maintaining a consistency and points that we have missed.